### PR TITLE
V1 migration

### PR DESCRIPTION
**Pull request by oipat**
Migrated to support Serverless 1.0.0.beta.2

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -23,22 +24,19 @@ npm install -g serverless
...
 ```
 
-* Initialize project and follow instructions
-```json
-serverless project init
-```
+* It not already configured, configure your AWS provider credentials as described here: https://git.io/v64aN
 
-* Delpoy your's lambdas and enpoints
+* Delpoy your serverless service
```
Comment (laardee) [inline README.md]: While editing the readme file, "Delpoy" could be changed to "Deploy".

File: package.json [hunk 2]
```diff
@@ -26,9 +26,16 @@
   },
   "homepage": "https://github.com/SC5/aws-serverless-hackathon-backend#readme",
   "dependencies": {
-    "serverless-cors-plugin": "^0.4.1"
+    "aws-sdk": "^2.2.48"
```
Comment (laardee) [inline package.json]: Why this is moved to dependencies?

File: test/createPost.js [hunk 3]
```diff
@@ -0,0 +1,35 @@
...
+// Generated by serverless-mocha-plugin
+
+const mod         = require('../blog/posts/handler.js');
+const mochaPlugin = require('serverless-mocha-plugin');
+const lambdaWrapper     = mochaPlugin.lambdaWrapper;
+const expect      = mochaPlugin.chai.expect;
+
+const liveFunction = {
+  region: process.env.SERVERLESS_REGION,
+  lambdaFunction: process.env.SERVERLESS_PROJECT + '-handler'
```
Comment (laardee) [inline test/createPost.js]: You could use string interpolation

**Review by laardee — COMMENTED** [review]
You are mixing single and double quotes in test. Also some semicolons are missing.

File: test/createPost.js (hunk 3 above)
Comment (oipat) [reply test/createPost.js]: I'll try to fix these soon. I am currently occupied with something else.

**Comment by laardee** [comment]
Nice! 👍

**Pull request merged**